src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java

```diff
@@ -372,8 +372,15 @@ private void startInternal()
         catch (Exception e)
         {
             logger.error("Failed to send start Jibri IQ: " + e, e);
-
-            throw new StartException(StartException.INTERNAL_SERVER_ERROR);
+            jibriDetector.memberHadTransientError(jibriJid);
+            if (!maxRetriesExceeded())
+            {
+                retryRequestWithAnotherJibri();
```

Comment:
Note that this is effectively a recursive call, as this calls `start` which calls `startInternal`.  We did not recursively call `start` before because `retryRequestWithAnotherJibri` was called in the context of handling a Jibri response.  I don't think this should be a problem (I'll be testing to verify that) but thought it was worth pointing out.